### DISCUSSION TITLE
Dual extruder upgrade

### DISCUSSION THREAD

**Discussion by Luk3hutchinson** [NARUTOfzr/Neptune_3#44] [Ideas]
Has anyone done a dual extruder on this machine yet?

**Comment by mlee12382** [comment]
u/BambusBo on reddit did it :) he's running on klipper but I have compiled marlin firmware based on the stock that enables it also.

**Comment by Luk3hutchinson** [comment]
Will take look really interested in using pva for soluable supports to save on materials and easy removal of supports

**Reply by mlee12382** [reply]
That's problematic, without a proper purge you will almost definitely have pva contamination in you pla which can cause problems with your print. That type of thing is best done with an idex setup.

**Reply by Luk3hutchinson** [reply]
Best option may be to tinker the supports with the PLA for now. But if different pla colours definitely something to keep in mind

**Reply by mlee12382** [reply]
Multicolor is a lot of fun :) once you have your support distance figured out so you're getting decent support but easy separation you can do a lot more complex prints. Getting your bridging settings dialed in is helpful also so you don't need as many supports to start with.
